pims/utils/sort.py

```diff
@@ -0,0 +1,31 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+from __future__ import division
+from __future__ import absolute_import
+from __future__ import print_function
```

Comment:
Thanks, @hadim.

I would prefer

    from __future__ import (absolute_import, division, print_function,
                            unicode_literals)

if you have time to push a quick revision.

This might be useful for custom readers, so I am OK with putting it `utils` where others can import it for use in their own readers.